federatedscope/cl/dataloader/Cifar10.py

```diff
@@ -0,0 +1,190 @@
+    )
+
+    splits = config.data.splits
+    path = config.data.root
+    name = config.data.type.upper()
+    client_num = config.federate.client_num
+    batch_size = config.data.batch_size
+    
+    data_train = CIFAR10(path, train=True, download=True, transform=transform_train)
+#     data_val = CIFAR10(path, train=True, download=True, transform=transform_train)
+    data_test = CIFAR10(path, train=False, download=True, transform=transform_train)
+    
+          # Split data into dict
+    data_dict = dict()
+
+    # Splitter
+    splitter = get_splitter(config)
+    data_train = splitter(data_train)
+    data_val = data_train
+    data_test = splitter(data_test)
+
+
+    client_num = min(len(data_train), config.federate.client_num
+                     ) if config.federate.client_num > 0 else len(data_train)
+    config.merge_from_list(['federate.client_num', client_num])
+
+    
+    for client_idx in range(1, client_num + 1):
+        dataloader_dict = {
+              'train':
+              DataLoader(data_train[client_idx - 1],
+                         config.data.batch_size,
+                         shuffle=config.data.shuffle,
+                         num_workers=config.data.num_workers),
+              'val':
+              DataLoader(data_val[client_idx - 1],
+                         config.data.batch_size,
+                         shuffle=False,
+                         num_workers=config.data.num_workers),
+              'test':
+              DataLoader(data_test[client_idx - 1],
+                         config.data.batch_size,
+                         shuffle=False,
+                         num_workers=config.data.num_workers),
+          }
+        data_dict[client_idx] = dataloader_dict
+    r"""
+
+    Returns:
+            data:
+                {
+                    '{client_id}': {
+                        'train': Dataset or DataLoader,
+                        'test': Dataset or DataLoader,
+                        'val': Dataset or DataLoader
+                    }
+                }
+            config:
+                cfg_node
+    """
+    config = config
+    return data_dict, config
+
+def Cifar4LP(config):
+    
+    transform_train = T.Compose([
+            T.RandomResizedCrop(32, scale=(0.5, 1.0), interpolation=T.InterpolationMode.BICUBIC),
+            T.RandomHorizontalFlip(p=0.5),
+            T.ToTensor(),
+            T.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))
+        ])
+    transform_test = T.Compose([
+        T.ToTensor(), 
+        T.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))]
+    )
+
+    
+    data_train = CIFAR10(config.data.root, train=True, download=True, transform=transform_train)
+    data_val = CIFAR10(config.data.root, train=True, download=True, transform=transform_test)
+    data_test = CIFAR10(config.data.root, train=False, download=True, transform=transform_test)
+    
+          # Split data into dict
+    data_dict = dict()
+
+    # Splitter
```

Comment:
Please refer to `splitter`https://github.com/alibaba/FederatedScope/blob/dfcfff3b41fc0bd2f8f2d3bd0b004618cd41ad52/federatedscope/core/auxiliaries/data_builder.py#L493 and make it consistant.